test/rpc/test_deps.py

```diff
@@ -72,42 +72,80 @@ def deps_with_packages(packages, bad_packages, project_dir, profiles_dir, schema
         querier.is_result(querier.async_wait(tok1))
 
 
+@pytest.mark.parametrize(
```

Comment:
@jtcohen6 I parametrized the test instead of the copy paste fyi

(I did this when I merged the changes from the PR that adds support for commits here)